octoprint_Spoolman/modules/PrinterHandler.py

```diff
@@ -43,6 +43,9 @@ def handlePrintingStatusChange(self, eventType):
         ):
             self.lastPrintOdometer = None
             self.lastPrintOdometerLoad = None
+            
+            self._settings.set([SettingsKeys.DATA_SAVE], None)
+            self._settings.save()
```

Comment:
`PRINT_PAUSED` event handling commits spool usage to the Spoolman server. This means that with current implementation of the backup feature, it is possible to double commit spool usage, because it does not take into account that some of the data might have been persisted already.